dlk/python/dlk/templates/src/func/impl/fpga/quantized_conv2d_kn2row.cpp

```diff
@@ -126,14 +126,9 @@ void QuantizedConv2DKn2Row(QUANTIZED_NOT_PACKED input[],
     const T_UINT k_size = k_n_aligend_with_num_pe * k_h * k_w * k_c_by_word;
 
     T_UINT kernel_hwnocni[k_size];
-    T_UINT kernel_filled_extra[k_size];
-
-    for (size_t k = 0; k < k_n * k_h * k_w * k_c_by_word; k++) {
-      kernel_filled_extra[k] = kernel[k];
-    }
 
     Measurement::Start("Kernel transpose NHWC to HWNoCNi");
-    kernel_transform_NHWC_to_HWNoCNi(kernel_filled_extra, kernel_hwnocni,
```

Comment:
I guess, the purpose of introducing additional buffer was that `k_n_aligend_with_num_pe` could be theoretically bigger than `k_n`, so it should prevent accessing of memory outside of `kernel[]` array.

At the moment everything is aligned at network architecture level anyway, so we probably will not see any difference. :thinking: 

BTW, Implementing #23 seems to be the best thing to do here, as it will
 - (a) make performance better
 - (b) allow us to keep implementation of `kernel_transform_NHWC_to_HWNoCNi()` as safe and readable as possible 
    - as we call it only once on the start, we do not need care about optimizing it right? :wink: